admin/css/debug-toolbar/toolbar.scss

```
@@ -510,3 +522,47 @@
 .debug-bar-noverflow {
     overflow: hidden;
 }
+
+.debug-bar-dtableRow{
+    display: table-row;
+  }
+  
+.debug-bar-dinlineBlock{
```

```suggestion
.debug-bar-dinlineBlock {
```
The same goes for the rest of the styles, please.